components/modal/modal.html

```diff
@@ -0,0 +1,105 @@
+            </a>
+        </nav>
+    </header>
+    <nav>
+        <ul class="sidebar-container">
+            <div class="heading-3">Guide</div>
+            <li class="sidebar-item"><a href="../../documentation/documentation.html" class="sidebar-link">How to
+                    use?</a></li>
+            <div class="sidebar-heading heading-3">Components</div>
+            <li class="sidebar-item"><a href="../../components/alerts/alert.html" class="sidebar-link">Alert</a></li>
+            <li class="sidebar-item"><a href="../../components/avatar/avatar.html" class="sidebar-link">Avatar</a></li>
+            <li class="sidebar-item"><a href="../../components/badge/badge.html" class="sidebar-link">Badge</a></li>
+            <li class="sidebar-item"><a href="../../components/button/button.html" class="sidebar-link">Button</a></li>
+            <li class="sidebar-item"><a href="../../components/card/card.html" class="sidebar-link">Card</a></li>
+            <li class="sidebar-item"><a href="../../components/image/image.html" class="sidebar-link">Image</a></li>
+            <li class="sidebar-item"><a href="../../components/input/input.html" class="sidebar-link">Input</a></li>
+            <li class="sidebar-item"><a href="../../components/lists/lists.html" class="sidebar-link">Lists</a></li>
+            <li class="sidebar-item"><a href="../../components/modal/modal.html" class="sidebar-link">Modal</a></li>
+            <li class="sidebar-item"><a href="../../components/navigation/navigation.html"
+                    class="sidebar-link">Navigation</a></li>
+            <li class="sidebar-item"><a href="../../components/text-utilities/text-utilities.html"
+                    class="sidebar-link">Text-utilities</a></li>
+        </ul>
+    </nav>
+
+    <main class="main-container">
+        <h2 class="heading-2 fw-regular m-tp-one">Modal</h2>
+        <hr>
+        <section class="section">
+            <p class="heading-4 m-tp-one fw-lght">To make a modal, copy paste the code provided.</p>
+            <div class="wrapper p-1 m-tp-one pos-rltv min-hght-20">
+                <button class="modal-show btn btn-solid-secondary">Show</button>
+                <div class="modal border-1px-solid pos-rltv z-index-three">
+                    <div class="modal-text heading-3"> This is a modal</div>
+                    <button class="modal-btn pos-absolute  m-r-1">
+                        <i class="modal-close far fa-times-circle"></i>
+                    </button>
+                </div>
+
+            </div>
+        </section>
+
+        <pre>
+            <code class="language-markup">
+                <div class="modal border-1px-solid pos-rltv z-index-three">
+                    <div class="modal-text heading-3"> This is a modal</div>
+                    <button class="modal-btn pos-absolute  m-r-1">
+                        <i class="modal-close far fa-times-circle"></i>
+                    </button>
+                </div>
```

Comment:
Why did you add this part in the `code` element?

Reply:
It's for rendering the code snippet on page.